commit 3d156389e4246afe059ba8a346de8e9a7c52c191
Author: Ozan Çağlayan <ozan@pardus.org.tr>
Date:   Thu Jan 19 15:59:55 2012 +0200

    cpupower: Fix linking with --as-needed
    
    Fix linking order to avoid undefined reference errors when
    using --as-needed linker flag.
    
    Signed-off-by: Ozan Çağlayan <ozan@pardus.org.tr>

diff --git a/tools/power/cpupower/Makefile b/tools/power/cpupower/Makefile
index e8a03ac..f01b3f5 100644
--- a/tools/power/cpupower/Makefile
+++ b/tools/power/cpupower/Makefile
@@ -193,7 +193,7 @@ $(UTIL_OBJS): $(UTIL_HEADERS)
 
 cpupower: $(UTIL_OBJS) libcpupower.so.$(LIB_MAJ)
 	$(ECHO) "  CC      " $@
-	$(QUIET) $(CC) $(CFLAGS) $(LDFLAGS) -lcpupower -lrt -lpci -L. -o $@ $(UTIL_OBJS)
+	$(QUIET) $(CC) $(CFLAGS) $(LDFLAGS) $(UTIL_OBJS) -lcpupower -lrt -lpci -L. -o $@
 	$(QUIET) $(STRIPCMD) $@
 
 po/$(PACKAGE).pot: $(UTIL_SRC)
